apps/api/admin.py

```diff
@@ -1,85 +1,170 @@
+'''Representation of the models in the admin interface'''
+
 from django.contrib import admin
 from apps.api import models
 
 
 class InstructorAdmin(admin.ModelAdmin):
-    pass
+    '''Adds instructor model to the admin interface
+
+    list_display: shows the columns that are present in the intructor interface
+    search_fields: shows the columns that can be searched for
+    ordering: how the information is sorted
+    '''
+    list_display = ['last_name', 'first_name', 'university_id', 'email']
+    search_fields = ['last_name', 'first_name', 'university_id', 'email']
+    ordering = ['last_name']
 
 
 class TeachingAssistantAdmin(admin.ModelAdmin):
-    pass
+    '''Adds teaching assistant model to the admin interface
+
+    list_display: shows the columns that are present in the teaching assistant
+        interface
+    search_fields: shows the columns that can be searched for
+    ordering: how the information is sorted
+    '''
+    list_display = ['last_name', 'first_name', 'university_id', 'email']
+    search_fields = ['last_name', 'first_name', 'university_id', 'email']
+    ordering = ['last_name']
 
 
 class IdentificationInline(admin.StackedInline):
+    '''Making it StackedInline means that this interface can also be
+    edited by other interfaces
+    '''
     model = models.Identification
 
 
+class IdentificationAdmin(admin.ModelAdmin):
```

Comment:
Many unnecessary admin classes that only passes.

Possible Solution: Remove this and only have admin.site.register(models.Identification) in this case. Modify accordingly for other models.